app/src/main/java/org/oppia/app/utility/RegionClickEvent.kt

```diff
@@ -10,5 +10,5 @@ sealed class RegionClickedEvent
  */
 data class NamedRegionClickedEvent(val regionLabel: String) : RegionClickedEvent()
 
-/** Class to be used in case when [OnClickableAreaClickedListener] is called with an unspecified region. */
-class DefaultRegionClickedEvent() : RegionClickedEvent()
+/** object to be used in case when [OnClickableAreaClickedListener] is called with an unspecified region. */
+object DefaultRegionClickedEvent : RegionClickedEvent()
```

Comment:
Suggest making this a data class instead of a singleton object--I think it will also work for your equals() check. I am curious why it was failing for you since it seemed to work for me with Robolectric.

Reply:
data class needs at least one primary constructor and it doesn't have any

Reply:
You can also use a non-data class and check the value using isInstanceOf instead of an equality check in the test. My concern is that this upsets the pattern for events: you create an event when you want to fire it, not reuse an existing event.